generators/app/templates/hooks/pre-commit

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
We're in agreement on the "gross" part, but it's more that I just don't like the idea of having code in our repo that we will never edit because we don't even know how to edit it. You're right that from a purely pragmatic standpoint it can probably exist in the repo and we never need to know that its there. But I _will_ know that its there, and it will eat me up inside. Oh well, I can live with it I guess.